viewing_party/party.py

```diff
@@ -1,23 +1,213 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
-    pass
+from symbol import single_input

```

Comment:
doesn't look like we are using this package anywhere. sometimes VSCode will try to be helpful and auto-import packages it thinks we are trying to type out. let's get rid of this

```suggestion

```